Docker/Dockerfile.simulation.ubuntu-galactic

```diff
@@ -0,0 +1,110 @@
+                       libglu1-mesa-dev \
+                       libxcb-xinerama0 \
+                       libxcb-xinput0 \
+                       libxcb-xinput-dev \
+                       libxcb-xfixes0-dev \
+                       libxcb-xkb-dev \
+                       libxkbcommon-dev \
+                       libxkbcommon-x11-dev \
+                       libfontconfig1-dev \
+                       libcurl4-openssl-dev \
+                       libsdl2-dev \
+                       zlib1g-dev \
+                       mesa-common-dev \
+                       libssl-dev libxcb-icccm4 \
+                       libxcb-image0 \
+                       libxcb-keysyms1 \
+                       libxcb-render-util0 \
+                       libxcb-randr0 \
+                       libnvidia-gl-470 \
+                       ufw
+
+# Add additional ROS2/Galactic packages
+RUN /bin/bash -c 'source /opt/ros/galactic/setup.bash && \
+                  apt install -y ros-${ROS_DISTRO}-slam-toolbox \
+                                 ros-${ROS_DISTRO}-navigation2 \
+                                 ros-${ROS_DISTRO}-nav2-bringup \
+                                 ros-${ROS_DISTRO}-pointcloud-to-laserscan \
+                                 ros-${ROS_DISTRO}-gazebo-msgs \
+                                 ros-${ROS_DISTRO}-ackermann-msgs \
+                                 ros-${ROS_DISTRO}-control-toolbox'
+
+
+# Clone O3DE repos, register, build, and cleanup in the same layer to reduce the size
+RUN cd $WORKSPACE && \
+    git clone --single-branch --recursive --branch development $O3DE_REPO && \
+    cd o3de && \
+    git lfs install && \
+    git lfs pull && \
+    cd $WORKSPACE/o3de && \
+    python/get_python.sh && \
+    cd $WORKSPACE/o3de && \
+    $WORKSPACE/o3de/scripts/o3de.sh register --this-engine && \
+    cd $WORKSPACE && \
+    git clone $ROS2_GEM_REPO --single-branch --branch development && \
+    $WORKSPACE/o3de/scripts/o3de.sh register -gp $WORKSPACE/o3de-ros2-gem && \
+    git clone --single-branch --recursive $LOFT_GEM_REPO && \
+    cd loft-arch-vis-sample && \
+    git lfs install && \
+    git lfs pull && \
+    $WORKSPACE/o3de/scripts/o3de.sh register -gp $WORKSPACE/loft-arch-vis-sample/Gems/ArchVis/ && \
+    cd $WORKSPACE && \
+    git clone --single-branch --branch main --recursive $O3DE_DEMO_REPO && \
+    cd o3de-demo-project && \
+    git lfs install && \
+    git lfs pull && \
+    $WORKSPACE/o3de/scripts/o3de.sh register -pp $WORKSPACE/o3de-demo-project/  && \
+    cd $WORKSPACE/o3de-demo-project && \
+    /bin/bash -c 'source /opt/ros/galactic/setup.bash && \
+                  cmake -B build/linux -S . -G "Ninja Multi-Config" -DLY_STRIP_DEBUG_SYMBOLS=TRUE -DLY_DISABLE_TEST_MODULES=ON && \
+                  cmake --build build/linux --config profile --target AssetProcessorBatch ROS2-Gem-Demo.GameLauncher ROS2-Gem-Demo.Assets && \
+                  cd $WORKSPACE && \
+                  ./cleanup.bash'
```

Comment:
You can change the directory for git command using the -`C` option (https://git-scm.com/docs/git#Documentation/git.txt--Cltpathgt)